Automattic/jetpack
# Docker: Set SCRIPT_DEBUG constant when setting up the environment

For many reasons this is useful but specially for developing React based interfaces.

Due to not having this set to on we have introduced several problems on these interfaces for long: See https://github.com/Automattic/jetpack/pull/22518,  https://github.com/Automattic/jetpack/pull/22524 and https://github.com/Automattic/jetpack/issues/22516, https://github.com/Automattic/jetpack/issues/22382, https://github.com/Automattic/jetpack/issues/22378 and https://github.com/Automattic/jetpack/issues/22380 for example

#### Changes proposed in this Pull Request:

* Makes the PHP constant SCRIPT_DEBUG  be set to true when setting up the docker dev environment


#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

For testing out you should destroy your current env. Maybe just check the change seems fine
